<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRasterLayerSaveAsDialogBase</class>
 <widget class="QDialog" name="QgsRasterLayerSaveAsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Raster Layer asâ€¦</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="1">
      <widget class="QgsFileWidget" name="mFilename" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mModeLabel">
       <property name="text">
        <string>Output mode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="mRawModeRadioButton">
         <property name="toolTip">
          <string>Write out raw raster layer data. Optionally user defined NoData values may be applied.</string>
         </property>
         <property name="text">
          <string>Raw data</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mRenderedModeRadioButton">
         <property name="toolTip">
          <string>Write out 3 bands RGB image rendered using current layer style.</string>
         </property>
         <property name="text">
          <string>Rendered image</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mFormatLabel">
       <property name="text">
        <string>Format</string>
       </property>
       <property name="buddy">
        <cstring>mFormatComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="mFormatComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mTileModeCheckBox">
         <property name="toolTip">
          <string>Create GDAL Virtual Format composed of multiple
datasets with maximum width and height specified below.</string>
         </property>
         <property name="text">
          <string>Create VRT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>CRS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mSaveAsLabel">
       <property name="text">
        <string>File name</string>
       </property>
       <property name="buddy">
        <cstring>mFilename</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Layer name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mLayerName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QgsScrollArea" name="mScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-157</y>
        <width>538</width>
        <height>602</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QgsExtentGroupBox" name="mExtentGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Extent</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mResolutionGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Resolution</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QRadioButton" name="mResolutionRadioButton">
              <property name="text">
               <string>Horizontal</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mXResolutionLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="mSizeRadioButton">
              <property name="text">
               <string>Columns</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mColumnsLineEdit"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="mRowsLabel">
              <property name="text">
               <string>Rows</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="mYResolutionLineEdit"/>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="mRowsLineEdit"/>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="mOriginalResolutionPushButton">
              <property name="text">
               <string>Layer Resolution</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="mOriginalSizePushButton">
              <property name="text">
               <string>Layer Size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Vertical</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mTilesGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>VRT Tiles</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="mMaximumSizeXLabel">
            <property name="toolTip">
             <string>Maximum number of columns in one tile.</string>
            </property>
            <property name="text">
             <string>Max columns</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mMaximumSizeXLineEdit">
            <property name="toolTip">
             <string>Maximum number of columns in one tile.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mMaximumSizeYLabel">
            <property name="toolTip">
             <string>Maximum number of rows in one tile.</string>
            </property>
            <property name="text">
             <string>Max rows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mMaximumSizeYLineEdit">
            <property name="toolTip">
             <string>Maximum number of rows in one tile.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mCreateOptionsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Create Options</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QgsRasterFormatSaveOptionsWidget" name="mCreationOptionsWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mPyramidsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Pyramids</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QgsRasterPyramidsOptionsWidget" name="mPyramidsOptionsWidget" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="mPyramidResolutionsLabel">
              <property name="text">
               <string>Resolutions</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="mPyramidResolutionsLineEdit">
              <property name="toolTip">
               <string>Pyramid resolutions corresponding to levels given</string>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="mPyramidsUseExistingCheckBox">
            <property name="text">
             <string>Use existing</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mNoDataGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Additional NoData values. The specified values will be set to NoData in output raster.</string>
         </property>
         <property name="title">
          <string>NoData values</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="1">
             <widget class="QPushButton" name="mRemoveSelectedNoDataToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Remove selected row</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="mAddNoDataManuallyToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Add values manually</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="5">
             <widget class="QTableWidget" name="mNoDataTableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>200</number>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>250</number>
              </attribute>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="mLoadTransparentNoDataToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Load user defined fully transparent (100%) values </string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/images/themes/default/mActionFileOpen.svg</normaloff>:/images/themes/default/mActionFileOpen.svg</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="mRemoveAllNoDataToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Clear all</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../images/images.qrc">
                <normaloff>:/images/themes/default/mActionRemove.svg</normaloff>:/images/themes/default/mActionRemove.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QDialogButtonBox" name="mHelpButtonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="mAddToCanvas">
       <property name="text">
        <string>Add saved file to map</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="mButtonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsRasterFormatSaveOptionsWidget</class>
   <extends>QWidget</extends>
   <header>qgsrasterformatsaveoptionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsRasterPyramidsOptionsWidget</class>
   <extends>QWidget</extends>
   <header>qgsrasterpyramidsoptionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mRawModeRadioButton</tabstop>
  <tabstop>mRenderedModeRadioButton</tabstop>
  <tabstop>mFormatComboBox</tabstop>
  <tabstop>mTileModeCheckBox</tabstop>
  <tabstop>mFilename</tabstop>
  <tabstop>mLayerName</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>mScrollArea</tabstop>
  <tabstop>mResolutionRadioButton</tabstop>
  <tabstop>mXResolutionLineEdit</tabstop>
  <tabstop>mYResolutionLineEdit</tabstop>
  <tabstop>mOriginalResolutionPushButton</tabstop>
  <tabstop>mSizeRadioButton</tabstop>
  <tabstop>mColumnsLineEdit</tabstop>
  <tabstop>mRowsLineEdit</tabstop>
  <tabstop>mOriginalSizePushButton</tabstop>
  <tabstop>mMaximumSizeXLineEdit</tabstop>
  <tabstop>mMaximumSizeYLineEdit</tabstop>
  <tabstop>mCreateOptionsGroupBox</tabstop>
  <tabstop>mPyramidsGroupBox</tabstop>
  <tabstop>mPyramidsUseExistingCheckBox</tabstop>
  <tabstop>mPyramidResolutionsLineEdit</tabstop>
  <tabstop>mNoDataGroupBox</tabstop>
  <tabstop>mNoDataTableWidget</tabstop>
  <tabstop>mAddNoDataManuallyToolButton</tabstop>
  <tabstop>mRemoveSelectedNoDataToolButton</tabstop>
  <tabstop>mLoadTransparentNoDataToolButton</tabstop>
  <tabstop>mRemoveAllNoDataToolButton</tabstop>
  <tabstop>mAddToCanvas</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
